.github/workflows/ecr-auth.yaml

```diff
@@ -0,0 +1,85 @@
+name: ECR Authentication test
+
+on:
+  pull_request_target:
+    branches: [ 'main' ]
+
+permissions:
+  # This let's us clone the repo
+  contents: read
+  # This let's us mint identity tokens.
+  id-token: write
+
+jobs:
+  krane:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCOUNT: 479305788615
+      AWS_REGION: us-east-2
+
+    steps:
+      - name: Set up Go 1.17.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.x
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Install krane
+        working-directory: ./cmd/krane
+        run: go install .
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@master
```

Comment:
Should have caught this: `aws-actions/configure-aws-credentials@v1.6.0` here and below. Dependabot should help us keep it up to date.

Reply:
https://github.com/google/go-containerregistry/pull/1238